test/io/ferguson/channel-comparision.good

```diff
@@ -0,0 +1,2 @@
+Channels a and b are equal.
+Channels a and b are not equal.
```

Comment:
This `.good` file probably needs a newline (though I haven't confirmed with testing)